docs/lookout-sdk.md

```diff
@@ -0,0 +1,119 @@
+# lookout-sdk Binary
+
+_For the **lookout-sdk** library to develop new analyzers go to [**lookout-sdk**](https://github.com/src-d/lookout-sdk) repository._
+
+`lookout-sdk` binary is a simplified version of the lookout server that works with a local git repository and does not need access to Github.
+
+You can think about `lookout-sdk` as a _curl-like_ tool to call an analyzer gRPC endpoints with a ReviewEvent or a PushEvent, from a local git repository, and send it to an analyzer without accessing GitHub at all. For convenience, `lookout-sdk` also exposes a **lookout DataService** backed by the same git repository.
+
+You can download the latest `lookout-sdk` from the [src-d/lookout releases page](https://github.com/src-d/lookout/releases).
+
+This is the sequence diagram of the `ReviewEvent` made by `lookout-sdk review`. You can compare it with a regular **lookout** installation at the [**source{d} lookout architecture** documentation](architecture.md).
+
+![sequence diagram](assets/lookout-sdk-seq-diagram.png)
```

Comment:
As I commented before. I don't like that a diagram shows `DataService` as a component, when it's just an internal part of the `lookout-sdk` command. Showing it at the same level as `lookout-sdk` may lead to believe you need to run something _else_, and it's not clear what.

I would merge `DataServer` with `lookout-sdk` in this diagram.

Reply:
addressed by 3484af7